### DISCUSSION TITLE
Add company logos

### DISCUSSION THREAD

**Discussion by dramseie** [Alex-D/Trumbowyg#1485] [Ideas]
I absolutely love this tool !
However, there is a Emoji and a Giphy plugin...
I think a company logo plugin would be great to have.
Any ideas ?

**Comment by Alex-D** [comment]
Like the image insert but with a predefined src + alt?

**Reply by dramseie** [reply]
Exactely :-)

**Reply by Alex-D** [reply]
Copy paste the plugin and edit it to put some logos as options and it's done :)
